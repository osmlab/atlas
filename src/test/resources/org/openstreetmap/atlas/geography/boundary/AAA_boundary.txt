AAA||POLYGON((27.236328125 26.255092778502082,31.015625 26.255092778502082,31.015625 23.544952263601573,27.236328125 23.544952263601573,27.236328125 26.255092778502082))#
