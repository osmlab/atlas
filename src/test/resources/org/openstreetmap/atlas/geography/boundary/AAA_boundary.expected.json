{
  "type": "FeatureCollection",
  "features": [
    {
      "type": "Feature",
      "geometry": {
        "type": "LineString",
        "coordinates": [
          [
            27.2363281,
            26.2550928
          ],
          [
            31.015625,
            26.2550928
          ],
          [
            31.015625,
            23.5449523
          ],
          [
            27.2363281,
            23.5449523
          ],
          [
            27.2363281,
            26.2550928
          ]
        ]
      },
      "properties": {
        "iso_country_code": "AAA"
      }
    }
  ]
}
